tests/conftest.py

```diff
@@ -880,3 +880,10 @@ def duthost_console(localhost, creds, request):
                        console_password=creds['console_password'][vars['console_type']])
     yield host
     host.disconnect()
+
+@pytest.fixture(scope='session')
```

Comment:
Sorry, should have asked this earlier. Can you add comments why this fixture is added. Since it is not autoused and it is not used in any current code, it might invite someone to 'clean it up'.

Reply:
I added a comment for you. Please review/edit as you see fit. Otherwise, I'll merge when test is done again.